### PR TITLE
EDM-620: enable using specific rpm releases in e2e testing

### PR DESCRIPTION
**Pull request by mangelajo**
Enables:
* testing of specific rpm builds via the `FLIGHTCTL_RPM` environment variable,
* the build of epel-9 as that is the repository pulled from the centos9 stream image.

In addition removes the "v" prefix from rpm builds, which is not standard via packit config change.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -228,6 +228,27 @@ export KUBEADMIN_PASS=your-oc-password-for-kubeadmin
 make in-cluster-e2e-test
 ```
 
+You can also use `FLIGHTCTL_RPM=release/0.3.0` or `FLIGHTCTL_RPM=devel/0.3.0-12-abcdefg`
```
Comment (mangelajo) [inline test/README.md:231]:
```suggestion
You can also use `FLIGHTCTL_RPM=release/v0.3.0.rc2-5.el9` or `FLIGHTCTL_RPM=devel/v0.3.0-12-abcdefg.el9`
```

**Review by Samudelacruz77 — APPROVED** [review]
LGTM

File: test/README.md [hunk 2]
```diff
@@ -228,6 +228,28 @@ export KUBEADMIN_PASS=your-oc-password-for-kubeadmin
...
 
+You can also use `FLIGHTCTL_RPM=release/0.3.0`, `FLIGHTCTL_RPM=devel/0.3.0.rc1-5.20241104145530808450.main.19.ga531984`
+or simply `FLIGHTCTL_RPM=release` or `FLIGHTCTL_RPM=devel` to consume an specific version/repository
+of the CLI and agent rpm.
+
+I.e. if you wanted to test the cluster along with the 0.3.0 release in
+https://copr.fedorainfracloud.org/coprs/g/redhat-et/flightctl/builds/, you would run:
+```bash
+export FLIGHTCTL_NS=flightctl
+export KUBEADMIN_PASS=your-oc-password-for-kubeadmin
```
Comment (siserafin) [inline test/README.md:239]: export KUBEADMIN_PASSWORD=your-oc-password-for-kubeadmin

Comment (siserafin) [reply test/README.md:239]: other than that /lgtm

Comment (mangelajo) [reply test/README.md:239]: ack, I will propose a follow-up PR to fix this thanks for catching it.

I want to get it merged for 0.3.0-rc3

**Pull request merged**